pull_alerts.py

```diff
@@ -87,6 +89,7 @@ def print_all_incidents(
         group_by_service
     )
     print_stats(all_incidents, include_stats)
+    # all_incidents = [i for i in all_incidents if 5 <= i.created_on.hour <= 15]
```

Comment:
do you need this commented out code?

Reply:
Ops, it's for local run to collect pages in my time zone, here by accident, I'll remove it